lib/Controller/PlayerSoftware.php

```diff
@@ -450,9 +465,12 @@ public function getSsspInstall(Request $request, Response $response)
 
             $versionInformation = $this->playerVersionFactory->getByMediaId($mediaId);
 
-            $this->outputSsspXml($versionInformation->version . '.' . $versionInformation->code, $media->fileSize);
+            $xml = $this->outputSsspXml($versionInformation->version . '.' . $versionInformation->code, $media->fileSize);
+            $response = $response
+                ->withHeader('Content-Type', 'application/xml')
+                ->write($xml);
         } else {
-            $response->withStatus(404);
+            return $response->withRedirect('/notFound');
```

Comment:
So I think this will cause an issue because Tizen requires a completely empty document with a 404 error code.